dictionary/README.md

```diff
@@ -3,3 +3,8 @@ Please write a couple of sentences explaining your choice of restart
 strategy.
 
 
+
+I choose permanent.
+First, in dictionary_requirement_3, if a random_word exits abnormally, it will restart automatically. So we should pick transient or permanent. I think permanent is better because it helps dictionary keep working even it exits normally but actually by mistake. As along as we playing the next game, we need dictionary again. So, dictionary should never exits actually.
```

Comment:
👍 

Most people chose :transient, but I agree with you 100%. :permanent expresses the intent (to keep the dictionary running) better than :transient.